### DISCUSSION TITLE
Windows Live Essentials Offline version

### DISCUSSION THREAD

**Discussion by Shaknunic** [LegacyUpdate/LegacyUpdate#17] [Ideas]
I wonder who would still use that, but in case someone do, the one offered on Legacy Update is the web version which means Microsoft servers should be working, since they pulled it off, It's impossible to download the required files, luckily it can be replaced by the offline version of the same program and that's what should be on Legacy Update tbh...

My system is in Portuguese Brazilian language so I found on archive this version of the program, It's the official copy of the offline installer: https://archive.org/details/wlsetup-all_ptbr

**Comment by jamesaepp** [comment]
#409 may be of help
